VTTiny/Editor/EditorGUI.cs

```diff
@@ -244,28 +246,32 @@ public static bool ActorDropdown(Scenery.Stage stage, Scenery.StageActor current
             newAsset = default;
 
             currentAsset?.RenderAssetPreview();
-            if (ImGui.BeginCombo($"{label}##AssetDropdown", $"{currentAsset?.Name ?? "No asset selected."}"))
+            if (!ImGui.BeginCombo($"{label}##AssetDropdown", $"{currentAsset?.Name ?? "No asset selected."}"))
             {
-                if (ImGui.Selectable("None", currentAsset == null))
-                {
-                    ImGui.EndCombo();
-                    return true;
-                }
+                return false;
+            }
 
-                foreach (var asset in database.GetAllAssetsOfType<T>())
-                {
-                    if (ImGui.Selectable(asset.Name ?? asset.Id.ToString(), asset == currentAsset))
-                    {
-                        ImGui.EndCombo();
+            if (ImGui.Selectable("None", currentAsset == null))
+            {
+                ImGui.EndCombo();
+                return true;
+            }
 
-                        newAsset = asset;
-                        return true;
-                    }
+            foreach (var asset in database.GetAllAssetsOfType<T>())
+            {
+                if (!ImGui.Selectable(asset.Name ?? asset.Id.ToString(), asset == currentAsset))
+                {
+                    continue;
```

Comment:
For if, or for(each) statements whose body only takes up one line, please remove the parentheses before and after and keep it to only one indented line. Thanks!